cli/cdsctl/main.go

```diff
@@ -63,15 +66,55 @@ func main() {
 		client = cdsclient.New(*cfg)
 	}
 
+	// hidden command, generateDocumentation, only used to generate hugo documentation
+	// run with ./cdsctl generateDocumentation
+	// souhld be
+	args := os.Args[1:]
```

Comment:
you can do hidden command with cobra, like this https://github.com/ovh/cds/blob/master/cli/cds/application/metadata.go#L44

Reply:
I'll fix that, thanks.